Yoast/YoastSEO.js
# Introduces Sentence Part object and English Participle object

## Relevant technical choices:

* Introduce an object for handling sentence parts.
* Introduce a language-specific participle object for English.

## Test instructions

This PR can be tested by following these steps:

* Make sure all tests pass.


Please make sure #921 is merged first!
